server/models/Document.ts

```diff
@@ -603,17 +616,22 @@ class Document extends ParanoidModel {
     this.createdById = userId;
     this.lastModifiedById = userId;
     this.publishedAt = null;
+    // Q: should it be detached from collection?
+    this.collectionId = null;
```

Comment:
I hope this is fine?(TODO: add tests if approved)

Reply:
Is it neccessary?

Reply:
No, not really in the current scope.